createdisk-library.sh

```diff
@@ -203,7 +203,7 @@ function install_additional_packages() {
 function prepare_hyperV() {
     local vm_ip=$1
 
-    install_additional_packages ${vm_ip} hyperv-daemons
+    ADDITIONAL_PACKAGES+=" hyperv-daemons"
```

Comment:
This new global variable looks more error-prone than explicit calls to `install_additional_packages` with the list of packages to install. If you modify `ADDITIONAL_PACKAGES` after `install_additional_packages` has been called, the package you wanted won’t be installed for no obvious reason.

Reply:
I didn't find any better way, may be have it local and then add packages to it and as part of install_additional_packages pass that list of packages. I wanted to run this install_additional_packages only once because it add a overlay layer each time so better to have all the changes in single layer.

Reply:
I suspected this kind of reasons but couldn’t see it in the commit log.
Maybe add a comment right after the call to `install_additional_packages` which says `Beyond this point, packages added to the ADDITIONAL_PACKAGES variable won’t be installed in the guest` or so?

Reply:
Added